validator/db/kv/attester_protection.go

```diff
@@ -78,38 +78,47 @@ func (store *Store) CheckSlashableAttestation(
 			return nil
 		}
 		// Check for surround votes.
-		return sourceEpochsBucket.ForEach(func(sourceEpochBytes []byte, targetEpochBytes []byte) error {
+		return sourceEpochsBucket.ForEach(func(sourceEpochBytes []byte, targetEpochsBytes []byte) error {
 			existingSourceEpoch := bytesutil.BytesToUint64BigEndian(sourceEpochBytes)
-			existingTargetEpoch := bytesutil.BytesToUint64BigEndian(targetEpochBytes)
-			existingAtt := &ethpb.IndexedAttestation{
-				Data: &ethpb.AttestationData{
-					Source: &ethpb.Checkpoint{Epoch: existingSourceEpoch},
-					Target: &ethpb.Checkpoint{Epoch: existingTargetEpoch},
-				},
-			}
-			// Checks if the incoming attestation is surrounding or
-			// is surrounded by an existing one.
-			surrounding := slashutil.IsSurround(att, existingAtt)
-			surrounded := slashutil.IsSurround(existingAtt, att)
-			if surrounding {
-				slashKind = SurroundingVote
-				return fmt.Errorf(
-					surroundingVoteMessage,
-					att.Data.Source.Epoch,
-					att.Data.Target.Epoch,
-					existingSourceEpoch,
-					existingTargetEpoch,
-				)
+
+			// There can be multiple target epochs attested per source epoch.
+			attestedTargetEpochs := make([]uint64, 0)
```

Comment:
we should be able to define the size upfront no? 

`len(targetEpochsBytes)/8`?

Reply:
Wouldn't it be len(targetEpochsBytes)?

Reply:
It would be len / 8, but then I would need to add some other counter variable given the for loop increments in 8s and not 1s

Reply:
It's not a big deal here, I don't foresee that many appends unless we have many epochs since finality, and even then, it won't be a bottleneck

Reply:
ah i thought it was bytes to bytes for some reason. Thanks!